src/bindings/js/node/src/compiled_model.cpp

```diff
@@ -122,3 +125,33 @@ Napi::Value CompiledModelWrap::export_model(const Napi::CallbackInfo& info) {
     const auto& exported = _stream.str();
     return Napi::Buffer<const char>::Copy(info.Env(), exported.c_str(), exported.size());
 }
+
+Napi::Value CompiledModelWrap::set_property(const Napi::CallbackInfo &info) {
+    Napi::Env env = info.Env();
+    try{
+        if (info.Length() != 1 || !info[0].IsObject()) {
+            OPENVINO_THROW("Expected a single object argument for setting properties");
+        }
+        const auto properties = to_anyMap(env, info[0]);
+        _compiled_model.set_property(properties);
+    } catch (const std::exception& e) {
+            reportError(env, e.what());
+    }
+    return env.Undefined();
+}
+
+Napi::Value CompiledModelWrap::get_property(const Napi::CallbackInfo &info) {
+    Napi::Env env = info.Env();
+    try {
+        if (info.Length() != 1 || !info[0].IsString()) {
```

Comment:
```suggestion
        if (ov::js::validate<Napi::String>(info, allowed_signatures)) {
```
ov::js::validate method will check the type of argument list, its length and provides unified error messages across  ov Node.js API